kectl/app/cmd/util/cloudinstaller.go

```diff
@@ -0,0 +1,261 @@
+package util
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"time"
+)
+
+type KubeCloudInstTool struct {
+	Common
+}
+
+func (cu *KubeCloudInstTool) InstallTools() error {
+	cu.SetOSInterface(GetOSInterface())
+	cu.SetKubeEdgeVersion(cu.ToolVersion)
+
+	err := cu.InstallKubeEdge()
+	if err != nil {
+
+		return err
+	}
+	fmt.Println("Installation of kubeedge package is sucessfull")
+	err = cu.generatecertificates()
```

Comment:
Can we come up with a phase for every step and the code related to that goes in that phase go file? 

see https://github.com/kubernetes/kubernetes/tree/master/cmd/kubeadm/app/phases for reference.

Reply:
https://github.com/srivatsav123/kubeedge/tree/kubeedgeinstaller/kubeedgeinst/cmd/phases/workflow
Actually if u see my skeleton code....I had added phases(I had a  thought on it)...Sameer had refractored the frame work, so i am going with the current implementation ...Will discuss with sameer about it..